companion/lib/Surface/Controller.js

```diff
@@ -26,6 +26,7 @@ import { getStreamDeckDeviceInfo } from '@elgato-stream-deck/node'
 import { usb } from 'usb'
 // @ts-ignore
 import shuttleControlUSB from 'shuttle-control-usb'
+import vecFootpedal from 'vec-footpedal'
```

Comment:
This causes:

> lib/Surface/Controller.js:29:26 - error TS7016: Could not find a declaration file for module 'vec-footpedal'. '/home/runner/work/companion/companion/node_modules/vec-footpedal/index.js' implicitly has an 'any' type.
>   Try `npm i --save-dev @types/vec-footpedal` if it exists or add a new declaration (.d.ts) file containing `declare module 'vec-footpedal';`
> 
> 29 import vecFootpedal from 'vec-footpedal'
>                             ~~~~~~~~~~~~~~~
> 
> 
> Found 1 error in lib/Surface/Controller.js:29

But I can't work out how e.g. Contour Shuttle doesn't cause the same thing...

Reply:
The `// @ts-ignore` on line 27 (the line before the import) is what allows it to work.
It would be really good to have typings enabled, but I guess that isn't required for this PR seeing as that one doesn't have it either.
```suggestion
// @ts-ignore
import vecFootpedal from 'vec-footpedal'
```

Reply:
Great thanks, couldn't see the wood for the trees!

If there's an example typing (for a non TS library) I can steal I can add one for one or indeed both in due course (but yes ideally after this PR is in). Presumably ideally the typing gets added to the source package so everyone benefits?

Reply:
Maybe https://github.com/bitfocus/node-infinitton-idisplay/blob/infinitton-idisplay/index.d.ts or https://github.com/node-hid/node-hid/blob/master/nodehid.d.ts are useful examples?
Ideally the `EventEmitter` would be typed too, I think that can be done without a library now, but I haven't tried it